dist/about.html

```diff
@@ -67,158 +67,179 @@ <h1 class="page-header">About</h1>
+					<strong>Background.</strong> The rough idea behind ToS;DR emerged during the <a href="https://events.ccc.de/camp/2011/wiki/ToS;DR" target="_blank" rel="nofollow">2011 Chaos Communication Camp</a> near Berlin, with people from <a href="https://unhosted.org" target="_blank" rel="nofollow">Unhosted</a> a movement to create web apps that give users control over their valuable user data and privacy. In the summer of 2012, <a href="https://hugoroy.eu/index.en.html" target="_blank" rel="nofollow">Hugo Roy</a>
+					(<a href="https://twitter.com/hugoroyd" target="_blank" rel="nofollow"><strong>@hugoroyd</strong></a>) started the legal analysis and brought the project to life. Since then, more people have joined the team and have contributed through the reviewer community. Ultimately, all the work is transparent and the discussions happen in public. Our work is <a href="thanks.html" target="_blank">funded</a> by non-profits organisations and individual donations and gets released as <a href="http://fsfe.org/about/basics/freesoftware.html">free software</a> and open data.</a>
 				</p>
 				<p>
-					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html">our classification</a>.
+					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html" target="_blank">our classification</a>.
 				</p>
 				<p>
 					<strong>What does “ToS;DR” mean?</strong> the name is inspired by internet acronym TL;DR which stands for "Too Long; Didn't Read" and is often used on blogs and emails when a block of text is just really long and that people are too lazy to read the whole stuff. It was intended more as a code name than as a real name. But it seems that people like it (do you?)
 				</p>
 				<p>
-					<strong><a href="contribute.html" class="btn btn-big btn-primary">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
+					<strong><a href="contribute.html" class="btn btn-big btn-primary" target="_blank">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
```

Comment:
```suggestion
					<strong><a href="contribute.html" class="btn btn-big btn-primary">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
```

Same goes for this one, its internal, no need for blanks